source/api_reference/version-history/index.html.md.erb

```diff
@@ -53,6 +53,17 @@ You can use the `/blobs` endpoint to retrieve multiple blobs in a single request
 
 Blobs are returned in the order they were added to the register. This guarantees that complete pages will always contain the same blobs.
 
+### New `/context` endpoint
+The context endpoint replaces `/register` ([RFC 18](https://github.com/openregister/registers-rfcs/blob/master/content/context-resource/index.md)).
+
+This endpoint exposes extra metadata about the schema of a register. Each register field is now an object containing `id`, `datatype` and `cardinality` attributes.
```

Comment:
I'm just slightly confused about "field" in the context of this sentence:

> Each register field is now an object containing `id`, `datatype` and `cardinality` attributes.

Since it seems like this PR is pushing for use of the term "attribute". I'm probably missing something so don't want to hold up merging.